### ISSUE TITLE
Can a OpenSSL release that contains PR's that exist in master but not in the 3.0.X  branch be created?

### ISSUE THREAD

**Issue by slontis** [openssl/openssl#18846]
If there was a 3.1 branch that was based off of 3.0 then it may need to consider some of the following differences.

(The list is still a work in progress - but does give an indication of the changes)..

Using a filter search of
````
is:pr  is:closed label:"branch: master" -label:"branch: 3.0" 
````

**Performance Regressions:**

https://github.com/openssl/openssl/pull/17862 Fix IV length caching in EVP encryption code
https://github.com/openssl/openssl/pull/17881 Refactor OSSL_LIB_CTX to avoid using CRYPTO_EX_DATA
https://github.com/openssl/openssl/pull/17921 Decoder resolution performance optimizations
https://github.com/openssl/openssl/pull/18105  evp_md_init_internal: Avoid reallocating algctx if digest unchanged
https://github.com/openssl/openssl/pull/18267 Make IV/buf in prov_cipher_ctx_st aligned
https://github.com/openssl/openssl/pull/18341 ossl_namemap_name2_num: Avoid OPENSSL_strndup()
https://github.com/openssl/openssl/pull/18354 improve ossl_lh_strcasehash

**Other Performance**
https://github.com/openssl/openssl/pull/17544 Simpler square-root computation for Ed25519
https://github.com/openssl/openssl/pull/17325 property: use a stack to efficiently convert index to string

**Other things that need to be considered:**
https://github.com/openssl/openssl/pull/16760 Change the default Security Level to 2
https://github.com/openssl/openssl/pull/17153 check the return value of EVP_MD_fetch in ecdh_exch.c & dh_exch.c

https://github.com/openssl/openssl/pull/17474 Use ERR upcalls in legacy provider
https://github.com/openssl/openssl/pull/17733  enable CMS sign/verify for provider-implemented PKEYs

https://github.com/openssl/openssl/pull/17873 Fix bug in scrypt KDF provider dup method (There is no dup in the 3.X branch – should there be?)
https://github.com/openssl/openssl/pull/17572 Add context dup functions for KDFs
https://github.com/openssl/openssl/pull/17529 Add missing cipher and digest context dup functions
https://github.com/openssl/openssl/pull/17997 OSSL_PARAM_get_*_ptr: Drop errors from ptr/string mismatch
https://github.com/openssl/openssl/pull/18054 s390: Add new machine generation
https://github.com/openssl/openssl/pull/18127 Fix gcc 6.3 builds of aarch64 BSAES
https://github.com/openssl/openssl/pull/18131 Move types.h #undefs for wincrypt.h compatibility
https://github.com/openssl/openssl/pull/18224/ evp_md: assert digest is provided for algctx reuse
https://github.com/openssl/openssl/pull/18236 tls: ban SSL3, TLS1, TLS1.1 and DTLS1.0 at security level one and above (this would not cherry pick nicely).

https://github.com/openssl/openssl/pull/18627 put_str: Use memcpy instead of strncpy (fix for latest gcc)

**Probably not these Features:??**
https://github.com/openssl/openssl/pull/16594 Keccak
https://github.com/openssl/openssl/pull/18794 Keccak implementation fix for s390

**Comment by beldmit** [comment]
https://github.com/openssl/openssl/issues/16791 also contains links to some performance improvements, but the first glance looks like they are covered.

**Comment by nhorman** [comment]
marking as inactive, will be closed at the end of 3.4 dev barring further discussoins
